expression/builtin_math_vec.go

```diff
@@ -365,3 +365,21 @@ func (b *builtinPowSig) vecEvalReal(input *chunk.Chunk, result *chunk.Column) er
 func (b *builtinPowSig) vectorized() bool {
 	return true
 }
+
+func (b *builtinAbsRealSig) vecEvalReal(input *chunk.Chunk, result *chunk.Column) error {
+	if err := b.args[0].VecEvalReal(b.ctx, input, result); err != nil {
+		return err
+	}
+	f64s := result.Float64s()
+	for i := 0; i < len(f64s); i++ {
+		if result.IsNull(i) {
```

Comment:
How about skipping this check(remove line 375-377), which may make it faster?

Reply:
before remove:
BenchmarkVectorizedBuiltinMathFunc/builtinAbsRealSig-VecBuiltinFunc-8         	  500000	      3157 ns/op	       0 B/op	       0 allocs/op
after remove:
BenchmarkVectorizedBuiltinMathFunc/builtinAbsRealSig-VecBuiltinFunc-8         	 2000000	       938 ns/op	       0 B/op	       0 allocs/op

so, should remove